django-backend/fecfiler/scha_transactions/models.py

```diff
@@ -74,7 +74,7 @@ class SchATransaction(SoftDeleteModel, CommitteeOwnedModel, ReportMixin):
     conduit_state = models.TextField(null=True, blank=True)
     conduit_zip = models.TextField(null=True, blank=True)
     memo_code = models.BooleanField(null=True, blank=True, default=False)
-    memo_text_description = models.TextField(null=True, blank=True)
```

Comment:
This will impact how the transaction will be serialized in the .fec.  (a design that is showing it's flaws by being brittle when we want to change the model).  You should leave this field as is and just don't populate it

Reply:
Oops, I had removed it and then *put it back*, but I guess I put it back the wrong way...